broker/broker_schema_mutator.go

```diff
@@ -31,12 +33,19 @@ type BrokerSchemaMutator struct {
 }
 
 func NewBrokerSchemaMutator() *BrokerSchemaMutator {
-	return &BrokerSchemaMutator{}
+	return &BrokerSchemaMutator{
+		tables: map[string]*memCom.TableSchema{},
+	}
 }
 
 // ====  metastore/common.TableSchemaMutator ====
-// TODO: implement. TableSchemaMutator should update b.tables
 func (b *BrokerSchemaMutator) ListTables() (tables []string, err error) {
+	tables = make([]string, len(b.tables))
+	i := 0
+	for _, table := range b.tables {
```

Comment:
```
for i, table := range b.tables {
		tables[i] = table.Schema.Name
	}
```

Reply:
lol can't remember why i didn't do that..

Reply:
oh I remembered. so b.tables is a map, i from the for iterator will be the key of string type.